### DISCUSSION TITLE
Create a manifest Gallery website

### DISCUSSION THREAD

**Discussion by ericsampson** [microsoft/winget-pkgs#3799] [General]
# Description of the new feature/enhancement
Having a public Gallery website as a user-friendly way to browse/search the manifests would be excellent. The `search` command in the CLI is a good start, but a website can be a lot richer experience, and some people would find it easier to use.
Examples:
- [NPM](npmjs.com)
- [NuGet](nuget.org)
- [chocolatey](chocolatey.org)
- [brew](brew.sh)
- many others...

Thanks!

**Comment by denelon** [comment]
Take a look at http://winstall.app/, https://winget.run/, or https://github.com/goranalkovic/WinGetGui. All three of those are community built UI mechanisms.

**Reply by ericsampson** [reply]
Thanks Demitrius. Would it be appropriate to add links to the first two sites on the [main WinGet docs site here](https://docs.microsoft.com/en-us/windows/package-manager/), with the appropriate caveats that they are not provided/endorsed by Microsoft. Otherwise discoverability is reduced significantly. But that raises questions around liability...

I still think it's worth having a 'blessed' Gallery website, whether that's community run or Microsoft run.
